.gitignore_template

```diff
@@ -0,0 +1,144 @@
+# Created by https://www.toptal.com/developers/gitignore/api/node
```

Comment:
Please just name this `.gitignore` not `.gitignore_template`. It should already be functional in any repo created from the template repo

Reply:
Done 👍